bigquery_etl/query_scheduling/templates/airflow_dag.j2

```diff
@@ -12,7 +12,7 @@ default_args = {{
 }}
 
 with DAG('{{ name }}', default_args=default_args{%+ if schedule_interval != None -%}, schedule_interval={{ schedule_interval | format_timedelta | format_schedule_interval }}{%+ endif -%}) as dag:
-{% for task in tasks %}
+{% for task in tasks | sort(attribute='task_name') %}
```

Comment:
Strangely, running the DAG generation in CircleCI puts the Airflow tasks in a completely different order. So, I think it makes sense to explicitly sort them here to avoid these issues.